src/datascience-ui/interactive-common/variableExplorer.tsx

```diff
@@ -498,4 +516,22 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
             this.props.showDataExplorer(row.buttons.variable, row.buttons.numberOfColumns);
         }
     };
+
+    private sortRows(sortColumn: string, sortDirection: 'ASC' | 'DESC' | 'NONE') {
```

Comment:
You should probably save the sortColumn/direction so that on refresh you can resort. Although that would mean sorting when set the state.

Reply:
Hmm, maybe not. Sorting would require fetching all rows so if you resorted on variable refresh that would likely make all variable refreshes fetch every row.

Reply:
Hmm, thinking about this more, it feels like the variables need to be sorted when requested from the kernel. This way you can still keep the paging algorithm

Reply:
And then you could save the sort column/direction, but instead of sorting in the UI, you'd make a new variable request to the server.

Reply:
Ahh, that makes sense. I'll try that. Thanks :)

Reply:
Something to keep in mind, the code you'll send to the kernel will likely be specific to python. Try to keep the python semantics out of the internal request as far down as you can go. This will make it easier to add support for say C#/Julia/R in the future. 

For example, you wouldn't want the sort column message from the UI to have any python specific stuff in it.